### DISCUSSION TITLE
QSelect's weird bug that would break the focus function (e.g. autofocus directive)

### DISCUSSION THREAD

**Discussion by guogangj** [quasarframework/quasar#11748] [General - Components / Directives / etc]
QSelect has a feature that handles the enter key. I found a bug about the enter key. Focus on a QSelect component then press the enter key very quickly, then the QSelect will show and hide the option list very quickly. It seems all right. However, that problem comes! You will find that autofocus will not work anymore until you refresh the whole page. Not only autofocus directive, but also any other functions relative to focus.

Briefly, the quickly enter key pressed on QSelect will break the browser's focus function.

I made a demo here: [codepen](https://codepen.io/guogangj/pen/PoJJYNX?editors=101)

At the beginning, use the toggle to show the input, the input will get focus. You can try multiple times. However, if you do "quickly enters on QSelect", then you will find the autofocus is broken.

----

Additional notes 1, I found it's not the enter key's problem, it should be the dropdown list's problem, I reproduced the problem with mouse operation.

Additional notes 2, I tried that at Chrome, Edge, Firefox, it acted same.

**Comment by pdanpdan** [comment]
I cannot reproduce it.
Can you please try to specify the exact steps to reproduce it?
Thank you

**Reply by guogangj** [reply]
https://user-images.githubusercontent.com/6103026/147167304-fbcfa4a5-30f9-4466-9249-b9eb59bdfebd.mp4

I made a short video for this.

**Reply by pdanpdan** [reply]
Thank you, it reproduces perfectly.

Please follow here: https://github.com/quasarframework/quasar/issues/11767
